src/plugins/ui_actions/public/components/action_wizard/index.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../../../../../src/legacy/ui/public/styles/_styling_constants';
+
+.uiActions__selectedActionFactoryContainer {
```

Comment:
Is this selector being used anywhere? I'm not seeing it in this PR.

Reply:
fixed